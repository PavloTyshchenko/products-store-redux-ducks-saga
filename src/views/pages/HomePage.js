import React from 'react';
import { Jumbotron, Button } from 'react-bootstrap';

const HomePage = props => {
    return (
        <Jumbotron>
            <h1>Hello there! Home Page</h1>
            <p>
                This is a simple react application to test myself,
                practice skills and obtain some new knowledge
            </p>
            <p>
                <Button variant="primary">Learn more</Button>
            </p>
        </Jumbotron>
    );
};

export default HomePage;
