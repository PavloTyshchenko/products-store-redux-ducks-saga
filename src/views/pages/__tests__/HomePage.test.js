import React from 'react';
import { shallow } from 'enzyme';

import HomePage from '../HomePage';


describe('HomePage Component', () => {

    it('Should render without errors', () => {
        const component = shallow(<HomePage />);
        expect(component.length).toBe(1);
    });

    test('Should match snapshot', () => {
        const component = shallow(<HomePage />);
        expect(component.debug()).toMatchSnapshot();
    })
});