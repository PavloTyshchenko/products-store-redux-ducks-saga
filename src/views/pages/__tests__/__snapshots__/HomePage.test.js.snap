// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component Should match snapshot 1`] = `
"<Bootstrap(Jumbotron)>
  <h1>
    Hello there! Home Page
  </h1>
  <p>
    This is a simple react application to test myself, practice skills and obtain some new knowledge
  </p>
  <p>
    <Button variant=\\"primary\\" active={false} disabled={false} type=\\"button\\">
      Learn more
    </Button>
  </p>
</Bootstrap(Jumbotron)>"
`;
